features/ContentCreation.feature

```diff
@@ -33,7 +33,7 @@ Scenario: Content can be previewed during creation
     And I set content fields
       | label | value               |
       | Title | Test Article2       |
-      | Intro | Test article2 intro |
+    And I set article intro "Test article2 intro"
```

Comment:
I'm not sure about intro, I'd go with something like "article main content field" (but shorter :D )